### PR TITLE
open-cluster-management/backlog#16250

### PR DESCRIPTION
**Pull request by swopebe**

### CODE REVIEW

File: release_notes/known_issues.adoc [hunk 1]
```diff
@@ -31,6 +31,16 @@ Review the known issues for {product-title}. The following list contains known i
 [#upgrade-known-issues]
 == Upgrade known issues
 
+[#upgrade-210-231-error]
+=== Upgrade from version 2.1.x to 2.3.2 degraded due to the Observability add-on
+// 2.1: 16250
+
+After you upgrade from 2.1.x to 2.3.2, some clusters might become degraded because the Observability add-on is not ready, or the image manifest ConfigMap is not read correctly during upgrade. which leads to an incorrect image.
```
Comment (almen-ng) [inline release_notes/known_issues.adoc]: There is an extra period after 
> ... during upgrade

Comment (almen-ng) [inline release_notes/known_issues.adoc]: > which leads to an incorrect image to be used

Comment (swopebe) [reply release_notes/known_issues.adoc]: changed to comma, as intended

**Review by almen-ng — APPROVED** [review]
Approved!

**Pull request merged**